app/dao/notifications_dao.py

```diff
@@ -519,3 +521,37 @@ def set_scheduled_notification_to_processed(notification_id):
         {'pending': False}
     )
     db.session.commit()
+
+
+def dao_get_total_notifications_sent_per_day_for_perfomance_platform(start_date, end_date):
+    """
+    SELECT
+    count(notifications),
+    coalesce(sum(CASE WHEN sent_at - created_at <= interval '10 seconds' THEN 1 ELSE 0 END), 0)
+    FROM notifications
+    WHERE
+    created_at > 'START DATE' AND
+    created_at < 'END DATE' AND
+    api_key_id IS NOT NULL AND
+    key_type != 'test' AND
+    notification_type != 'letter';
+    """
+    under_10_secs = Notification.sent_at - Notification.created_at <= timedelta(seconds=10)
```

Comment:
this function should be using `NotificationHistory` rather than `Notification` everywhere (including in the SQL docstring), because we don't want to count research mode notifications (they don't follow the normal queues so shouldn't affect performance), but there's currently no easy way of telling research mode notifications apart from real notifications, except that research mode notifications don't persist to the `NotificationHistory` table